docs/pbc/all/multi-factor-authentication/202505.0/howto-use-multi-factor-authentication-with-glue-api.md

```diff
@@ -0,0 +1,256 @@
+---
+title: How to Use Multi-Factor Authentication with Glue API
+description: Learn how to create and implement your own Multi-Factor Authentication method in Spryker.
+template: howto-guide-template
+last_updated: Apr 7, 2025
+related:
+  - title: Multi-Factor Authentication Feature overview
+    link: docs/pbc/all/multi-factor-authentication/page.version/multi-factor-authentication.html
+  - title: Install the Multi-Factor Authentication feature
+    link: docs/pbc/all/multi-factor-authentication/page.version/install-multi-factor-authentication-feature.html
+  - title: Install Customer Email Multi-Factor Authentication method
+    link: docs/pbc/all/multi-factor-authentication/page.version/install-customer-email-multi-factor-authentication-method.html
+  - title: Create Multi-Factor Authentication methods
+    link: docs/pbc/all/multi-factor-authentication/page.version/create-multi-factor-authentication-methods.html
+---
+
+This guide explains how to activate, deactivate, and use Multi-Factor Authentication (MFA) through Glue API. 
+
+To lean more about MFA methods, see [Multi-Factor Authentication feature overview](/docs/pbc/all/multi-factor-authentication/{{page.version}}/multi-factor-authentication.html).
+
+{% info_block infoBox "MFA Type Used in This Guide" %}
+
+This guide demonstrates the Multi-Factor Authentication (MFA) flow using the email MFA method. If you're implementing other MFA types, adapt the type field in the request payloads accordingly.
+
+{% endinfo_block %}
+
+## Prerequisites
+
+[Install the Multi-Factor Authentication feature](/docs/pbc/all/multi-factor-authentication/{{page.version}}/install-multi-factor-authentication-feature.html)
+[Install Customer Email Multi-Factor Authentication method](/docs/pbc/all/multi-factor-authentication/{{page.version}}/install-customer-email-multi-factor-authentication-method.html)
+
+## 1) Get Available MFA Methods
+
+To determine the user's current MFA status, request the list of available methods.
```

Comment:
please also explain that it will provide all MFAs that are available in the system and status for the customer